rest-api/src/test/java/gov/cms/qpp/conversion/api/acceptance/CpcApiAcceptance.java

```diff
@@ -21,18 +35,25 @@
 
 @ExtendWith(RestExtension.class)
 class CpcApiAcceptance {
-
 	private static final String CPC_UNPROCESSED_FILES_API_PATH = "/cpc/unprocessed-files";
 	private static final String CPC_FILE_API_PATH = "/cpc/file/";
 	private static final String PROGRAM_NAME_XPATH = "/*[local-name() = 'ClinicalDocument' and namespace-uri() = 'urn:hl7-org:v3']"
 		+ "/./*[local-name() = 'informationRecipient' and namespace-uri() = 'urn:hl7-org:v3']"
 		+ "/*[local-name() = 'intendedRecipient' and namespace-uri() = 'urn:hl7-org:v3']"
 		+ "/*[local-name() = 'id' and namespace-uri() = 'urn:hl7-org:v3'][@root='2.16.840.1.113883.3.249.7']/@extension";
 	private static final String CPC_PLUS_PROGRAM_NAME = "CPCPLUS";
-	private static final String NOT_A_CPC_FILE = "beed7ed4-107c-400f-b0de-0c60abc54344";
+	private static AmazonDynamoDB dynamoClient = AmazonDynamoDBClientBuilder.standard().build();
+	private static DynamoDBMapper mapper;
+	private static AWSKMS kmsClient = AWSKMSClientBuilder.defaultClient();
 
 	@BeforeAll
-	static void createUnprocessedItem() {
+	static void setUp() {
+		String kmsKey = "arn:aws:kms:us-east-1:850094054452:key/8b19f7e9-b58c-4b7a-8162-e01809a8a2e9";
```

Comment:
This will likely fail for IAM users who aren't privy.
```
$ aws kms encrypt --key-id arn:aws:kms:us-east-1:850094054452:key/8b19f7e9-b58c-4b7a-8162-e01809a8a2e --plaintext meepmawp

An error occurred (AccessDeniedException) when calling the Encrypt operation: 
```

Reply:
Yes. Unfortunately, we have to set up specific permissions for any user to run that specific kms key.

Reply:
I feel like this can be handled better in the light deployment. Instead of setting up another kms key on our AWS account for local testing.

Reply:
@saquino0827 @clydet Any traction on a better solution for this?  Or is the solution to just merge this, get the light deployment merged, and then modify afterwards?